pkg/backingstore/reconciler.go

```diff
@@ -890,6 +890,26 @@ func (r *Reconciler) reconcileExistingPods(podsList *corev1.PodList) error {
 	noneAttachingAgents := 0
 	failedAttachingAgents := 0
 	for _, pod := range podsList.Items {
+		for j := range pod.Spec.Containers.Containers[0].Env {
+			switch c.Env[j].Name {
+			case "HTTP_PROXY":
+				value := os.Getenv("HTTP_PROXY")
+				if value != c.Env[j].Value {
```

Comment:
not sure I understand the condition. you should use `LookupEnv`to set the correct value

Reply:
No really, `os.GetEnv` will return an empty string for empty or non-existing envs. This condition was set just because Jacky didn't want me to reset the envs in the case where the value didn't change, Which I understand is pointless, resetting to the same value will give the same result.